astro/src/content/blog/vatix-fusionauth.mdx

```diff
@@ -0,0 +1,52 @@
+---
+publish_date: 2024-10-31 
+title: Vatix Chooses FusionAuth to Streamline Authentication and Save Developer Resources
+description: What happens when your homegrown auth system is no longer sustainable?
+authors: Candace Werry
+image: /img/blogs/vatix/vatix-fusionauth.png
+tags: mfa, sso, enterprise
+categories: Community
+excerpt_separator: "{/* more */}"
+---
+
+Vatix needed a scalable, automated solution that could integrate with their enterprise customers' identity providers
+{/* more */}
+## THE CHALLENGE
```

Comment:
We don't have all caps on any other headers anywhere. Would make this normal case: `The Challenge`.